### DISCUSSION TITLE
请帮我退一下款我想先月订阅不想先年订阅

### DISCUSSION THREAD

**Discussion by destinymad** [community/community#73115] [Copilot Conversations]
### Select Topic Area

Show & Tell

### Body

请帮我退一下款我想先月订阅不想先年订阅

**Comment by ktauchathuranga** [comment]
open a ticket in here
https://support.github.com/request

**Reply by destinymad** [reply]
我已经这样写了 不知道对不对 
![image](https://github.com/community/community/assets/16535995/9e330727-1bdb-4834-a46d-174c82d551cc)

**Reply by destinymad** [reply]
![image](https://github.com/community/community/assets/16535995/99d1ccf9-66d6-4220-82e8-75d2f536bd5f)
有几个工单 但是一直没联系我

**Comment by ktauchathuranga** [comment]
i think that would be great if you can write that ticket in English...🖤
